README.md

```diff
@@ -94,11 +101,12 @@ To add a new service (see the [backend](./services/backend/) for an extensive ex
 2. call it as the service should be named
```

Comment:
this part wasn't changed by this commit but I think "name the service" would be easier to understand